go/vt/discovery/healthcheck_test.go

```diff
@@ -514,33 +584,35 @@ func discoveryDialer(tablet *topodatapb.Tablet, timeout time.Duration) (queryser
 
 type fakeConn struct {
 	queryservice.QueryService
-	tablet *topodatapb.Tablet
-	hcChan chan *querypb.StreamHealthResponse
-	errCh  chan error
+	tablet  *topodatapb.Tablet
+	hcChan  chan *querypb.StreamHealthResponse // next health response from the tablet
```

Comment:
nit: Can you please put the comment on the previous line instead? (This is also what our internal C++ style guide recommends ;)

Additionally, please follow the Go style for comments: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

i.e.

```go
// hcChan should be an unbuffered channel which holds the tablet's next health response.
hcChan  chan *querypb.StreamHealthResponse
```

and so on.